prototype2/gdgem/clustering/AbstractClusterer.h

```diff
@@ -11,6 +11,8 @@
 #include <gdgem/clustering/HitContainer.h>
 #include <gdgem/nmx/Event.h>
 
+namespace Gem {
+
 /// \todo pick a faster data structure
 using ClusterList = std::list<Cluster>;
```

Comment:
Unless you have proof to show that `std::list` gives you better performance in this specific case, you should really replace it with `std::vector`.

Reply:
This will soon get replaced by the generalized clustering classes, where this was likely fixed. This code is essentially deprecated.